src/components/Button.tsx

```diff
@@ -0,0 +1,24 @@
+type ButtonProps = {
+  text: string;
+  color: string;
+}
+
+export default function Button({ text, color }: ButtonProps) {
```

Comment:
Following standard convention, we use **Named exports for components**, **Default exports for pages** 
_(pages concept will be introduced later with Nextjs but since we're only using React we can just ignore that and use named exports for now)_

Also, we should opt for using **arrow functions** as this improves readability and follows [our React coding rules.](https://pj100.esa.io/posts/16606#%E3%82%A2%E3%83%AD%E3%83%BC%E9%96%A2%E6%95%B0%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%99%E3%82%8B)

So please change this component into named exports like this:

```tsx
export const Button: React.FC<ButtonProps>: ({text, color})
```

Reply:
Fixed